### PR TITLE
Optimize the logic of update Mappings

### PR DESCRIPTION
**Pull request by binyu1005**

### CODE REVIEW

File: arex-storage-web-api-service/src/main/java/com/arextest/storage/web/api/service/bean/StorageCacheConfiguration.java [hunk 1]
```diff
@@ -34,6 +39,20 @@ CacheProvider cacheProvider() {
...
+        }
+        Cache<String, String> cache = Caffeine
+                .newBuilder()
+                .initialCapacity(1000)
+                .maximumWeight(maximumWeight)
+                .weigher((String key, String value) -> key.length() + value.length())
+                .build();
+        return cache;
+    }
+
```
Comment (mokdeve) [inline arex-storage-web-api-service/src/main/java/com/arextest/storage/web/api/service/bean/StorageCacheConfiguration.java:55]: There is an instance of CacheProvider used by storage, Any others is unnecessary and redundant.

Comment (binyu1005) [reply arex-storage-web-api-service/src/main/java/com/arextest/storage/web/api/service/bean/StorageCacheConfiguration.java]: done.
Replacing local cache with Redis

File: arex-storage-core/pom.xml [hunk 2]
```diff
@@ -57,5 +57,9 @@
             <artifactId>google-collections</artifactId>
             <version>1.0</version>
         </dependency>
+        <dependency>
+            <groupId>com.github.ben-manes.caffeine</groupId>
```
Comment (mokdeve) [inline arex-storage-core/pom.xml:61]: Remove unused imports

File: arex-storage-core/src/main/java/com/arextest/storage/core/service/AgentWorkingService.java [hunk 3]
```diff
@@ -140,22 +145,32 @@ private <T extends MockItem> void updateMapping(@NotNull MockCategoryType catego
         if (item.getClass() == ServletMocker.class) {
             ServletMocker servlet = (ServletMocker) item;
 
-            ServiceEntity serviceEntity = serviceRepository.queryByAppId(servlet.getAppId());
-            if (serviceEntity == null) {
-                return;
+            String appServiceKey = SERVICE_MAPPINGS_PREFIX + servlet.getAppId();
```
Comment (mokdeve) [inline arex-storage-core/src/main/java/com/arextest/storage/core/service/AgentWorkingService.java:148]: A variable should be declared for "CacheKeyUtils.toUtf8Bytes(appServiceKey)" in order to create fewer Byte instances for next reuse.

**Review by mokdeve — APPROVED** [review]

**Pull request merged**